geomstats/geometry/grassmannian.py

```diff
@@ -52,12 +59,14 @@ def belongs(self, point, tolerance=TOLERANCE):
         Parameters
         ----------
         point : array-like, shape=[..., n, n]
+            Point to be checked.
         tolerance : int
-            default: TOLERANCE
+            Optional, default: TOLERANCE
```

Comment:
idem, this way it won't be possible what the default tol is from the doc

Reply:
Done.